# Request (empty)
---
# Response
Genotype[] genotypes
